release notes/v0.36.0.md

```diff
@@ -0,0 +1,135 @@
+
+## New Features!
+
+### Source Maps support ([#2082](https://github.com/grafana/k6/pull/2082))
+
+Following [#2082](https://github.com/grafana/k6/pull/2082), k6 now has support for [Source Maps](https://developer.mozilla.org/en-US/docs/Tools/Debugger/How_to/Use_a_source_map). k6 will try to load source maps either from the file system(s) or from inside the script, based on the standard(-ish) `//#sourceMappingURL=` comments. Furthermore, as k6 internally uses Babel to transform ES6+ scripts to ES5.1+, it will now make use of its ability to generate source maps, including combining with previously generated ones, to report correct line numbers. This should fix [#1804](https://github.com/grafana/k6/issues/1804); however, we anticipate some more issues will arise, and further tweaking will be necessary.
+
+Thus, given an `imported.js` module such as:
+```javascript
+export function f1() {
+  throw "line 2";
+}
+
+export function f2() {
+  throw "line 6";
+}
+
+export function f3() {
+  throw "line 10";
+}
+```
+
+and a k6 test script importing it as such:
+```javascript
+import { f2 } from "./imported.js"
+
+export default function() {
+  f2();
+}
+```
+
+Previous versions of k6 would report an error stack trace indicating an invalid line number in `imported.js` (*10*):
+```
+ERRO[0000] line 6
+        at f2 (file:///some/path/imported.js:10:61(2))
+        at file:///some/path/sourcemap.js:4:20(4) executor=per-vu-iterations scenario=default source=stacktrace
+```
+
+Starting with `v0.36.0` and source maps support, k6 would now report the exception at the correct line in `imported.js`:
+```
+ERRO[0000] line 6
+        at f2 (file:///some/path/imported.js:6:2(2))
+        at file:///some/path/loadtest.js:4:2(4)
+        at native executor=per-vu-iterations scenario=default source=stacktrace
+```
+
+#### Temporary warning
+
+Note that if a file size is greater than **250kb**, and the internal Babel is needed: Babel will not generate source map. This is because during internal testing it was found this takes *3x* to *4x* more memory, potentially leading to OOMs (Out Of Memory condition in which the OS kills a process for using too much memory) on bigger inputs. If required, you can control the accepted file size limit via the `K6_DEBUG_SOURCEMAP_FILESIZE_LIMIT` environment variable. A pre-generated source map will always be loaded. For more details, check [#2345](https://github.com/grafana/k6/pull/2345).
+
+### Ability to abort tests ([#2093](https://github.com/grafana/k6/pull/2093))
+
+Thanks to the contribution of [@gernest](https://github.com/gernest), k6 now has the ability to abort a test. [#2093](https://github.com/grafana/k6/pull/2093) made the `k6/execution` expose the `test.abort()` function, which allows to immediately abort a k6 load test run with the exit code `108`. Note that aborting a test is also fully supported in k6 cloud, preserving the same user-experience for both local and cloud users. 
```

Comment:
```suggestion
Thanks to the contribution of [@gernest](https://github.com/gernest) ([#2093](https://github.com/grafana/k6/pull/2093)), k6 now has the ability to abort a test run from within the test script. The newly added `test.abort()` function in the [`k6/execution` module](https://k6.io/docs/javascript-api/k6-execution/) allows k6 scripts to immediately abort the test execution - the VU that called it will abort immediately and any other VUs in the same or other instances (in the case of `k6 cloud`) will also be interrupted and abort soon after. Local `k6 run` tests will exit with a code of `108`, so this event can also be easily detected in a CI script. 
```

WDYT?